conda_forge_tick/migrators/cross_compile.py

```diff
@@ -212,3 +209,46 @@ def migrate(self, recipe_dir: str, attrs: "AttrsTypedDict", **kwargs: Any) -> No
 
             with open("build.sh", "w") as f:
                 f.write("".join(lines))
+
+
+class Build2HostMigrator(MiniMigrator):
+    post_migration = False
+
+    def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
+        build_reqs = attrs.get("requirements", {}).get("build", set())
+        host_reqs = attrs.get("requirements", {}).get("host", set())
+        if (
+            len(attrs.get("output_names", [])) <= 1
+            and build_reqs
+            and not host_reqs
```

Comment:
```suggestion
            and 'host' not in attrs.get("requirements", {})
```

Reply:
This change is wrong. The top-level requirements in the node attrs always has `host` even if host is empty. This is not the recipe requirements.

See here: 

https://github.com/regro/cf-graph-countyfair/blob/ddde68f0f61f1a8f2e54b865bb477716d460e125/node_attrs/jupyter_core.json#L1471

for example.

Reply:
woops resolved that too soon! sorry!

Reply:
I pushed a change just now that does what I think you were going for. It makes sure that the recipe doesn't have any host section as opposed to an empty one.

Reply:
Thanks

Reply:
resolving now!